src/expr/src/scalar/func.rs

```diff
@@ -5189,7 +5189,14 @@ fn list_slice<'a>(datums: &[Datum<'a>], temp_storage: &'a RowArena) -> Datum<'a>
     }
 
     temp_storage.make_datum(|row| {
-        slice_and_descend(datums[0], &ranges, row);
+        let empty_results = slice_and_descend(datums[0], &ranges, row);
+        // Empty results return an empty list and not NULL.
```

Comment:
What's the motivation for this change? Does it need to be described in release notes?

Reply:
This makes things behave like PG array for `slice_and_descend`; this should get called out in release notes because 1D slicing was _not_ gated behind the experimental flag, which I forgot. ty for the catch.